arch/x86/core/ia32/intstub.S

```diff
@@ -112,11 +112,12 @@ SECTION_FUNC(PINNED_TEXT, _interrupt_enter)
 	 * EAX = isr_param, EDX = isr
 	 */
 
-	/* Push EDI as we will use it for scratch space.
+	/* Push EBP as we will use it for scratch space.
```

Comment:
The changes to this file could probably be in another commit / PR to document this change better